src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/IfError.txt

```diff
@@ -149,6 +149,9 @@ Error({Kind:ErrorKind.InvalidArgument})
 >> IfError(1/0, Time(12,0,0))
 0.5
 
+>> IfError(1, 1/0, Time(12,0,0))
+0.5
```

Comment:
This currently returns Time(12,0,0,0) - a time value - not a number value.

Reply:
updated as discussed offline